docker-compose/docker-compose.yml

```diff
@@ -116,6 +116,8 @@ services:
     image: redis:2.8
     expose:
       - "6379"
+    ports:
+      - "6379:6379"  
```

Comment:
I can move this `PORT` into a separate PR, but the reason it exists is to provide a local cache for some actions (i.e. auth via PassportJS which uses https://github.com/ddragosd/openwhisk-cache-redis)